CHANGELOG.rst

```diff
@@ -23,6 +23,15 @@ Fixed
 Security
 ========
 
+[2022.1rc1] - "jovelina" - 2022-01-07
```

Comment:
@italovalcy @ajoaoff I've set `jovelina` as the release name as we've chatted, and also upated the version as `2022.1rc1`.